src/main/java/com/marklogic/kafka/connect/MarkLogicConfig.java

```diff
@@ -30,16 +31,28 @@ public class MarkLogicConfig extends AbstractConfig {
     public static final String SSL_HOST_VERIFIER = "ml.connection.customSsl.hostNameVerifier";
     public static final String SSL_MUTUAL_AUTH = "ml.connection.customSsl.mutualAuth";
 
-    private static final CustomRecommenderAndValidator SecurityContextTypeRV = new CustomRecommenderAndValidator("DIGEST", "BASIC", "CERTIFICATE", "KERBEROS", "NONE");
+    private static final CustomRecommenderAndValidator CONNECTION_HOST_RV = new CustomRecommenderAndValidator();
+    private static final CustomRecommenderAndValidator CONNECTION_PORT_RV = new CustomRecommenderAndValidator();
```

Comment:
For free-form String fields, I think we can just use the Validators that are included in the Kafka library.  Please take a look at the Validator for DSL_PLAN in MarkLogicSourceConfig.